test/app/dftb+/tests

```diff
@@ -18,6 +18,9 @@ deltadftb/C2H4_unpurified              #? OMP_THREADS <= 2 and not WITH_MPI
 deltadftb/indigo                       #? OMP_THREADS <= 4 and MPI_PROCS <= 2
 deltadftb/H2O                          #? OMP_THREADS <= 2 and not WITH_MPI
 
+timeprop/C4H6_rs                       #? OMP_THREADS <= 4 and not WITH_MPI
+timeprop/C4H6_rs_Singlet               #? OMP_THREADS <= 4 and not WITH_MPI
+#!timeprop/C4H6_rs_Triplet               #? OMP_THREADS <= 4 and not WITH_MPI
```

Comment:
The test is commented out, so should be harmless. Looking at the commit history it might be somewhat messy to remove it from the commits, but could be taken out with another commit on top of this (but remains in the logs).